src/index.html

```diff
@@ -7,6 +7,74 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div
+      class="
+        calendar
+        calendar--start-day-sun
+        calendar--months-length-31
+      "
+    >
+      <div class="calendar__day"></div>
+
+      <div class="calendar__day"></div>
+
+      <div class="calendar__day"></div>
+
+      <div class="calendar__day"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```
Newlines are redundant here

Reply:
And [in this example](https://github.com/mate-academy/layout_product-cards/pull/4065#discussion_r1475831915), newlines are also redundant?